.github/workflows/golangci-lint.yml

```
@@ -0,0 +1,28 @@
+jobs:
+  golangci:
+    name: Lint Go code with golangci-lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
```

actionlint (1.7.11)

[error] the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)